server/src/api/router.go

```diff
@@ -184,10 +184,11 @@ func (s *Server) protectedRoutes(r chi.Router) {
 
 func (s *Server) initVoteResources(r chi.Router) {
 	r.Route("/votes", func(r chi.Router) {
-		r.Use(s.BoardParticipantContext)
+		r.Use(s.BoardEditableContext)
+
 		r.Post("/", s.addVote)
 		r.Delete("/", s.removeVote)
-		r.Get("/", s.getVotes)
+		r.With(s.BoardParticipantContext).Get("/", s.getVotes)
```

Comment:
This solution isn't really readable in my opinion. The BoardParticipantContext basically tells me that for a specific endpoint, the requestor has to be a participant of the board. Now with this change I'm confused because it is not immediately recognisable to me that all endpoints require the participation of the board since only getVotes uses the BoardParticipantContext.

Maybe something like that would work:
```go
func (s *Server) initVoteResources(r chi.Router) {
	r.Route("/votes", func(r chi.Router) {
    r.Use(s.BoardParticipantContext)

		r.Get("/", s.getVotes)

    r.Group(func(r chi.Router) {
      r.Use(s.BoardEditableContext)

      r.Post("/", s.addVote)
      r.Delete("/", s.removeVote)
    })
	})
}

```

Reply:
You are correct! Changed it to your example.